javascript/README.md

```diff
@@ -0,0 +1,13 @@
+# Building the JavaScript libsbp bindings
+
+From the parent directory, run `make javascript` to regenerate bindings.
+
+# Using the JavaScript libsbp bindings
```

Comment:
yeah, I intend to do that - hopefully as part of the build process - once this PR has been approved